### DISCUSSION TITLE
Search: return different user names/IDs from a search of Facebook

### DISCUSSION THREAD

**Discussion by alessandro-buy** [kevinzg/facebook-scraper#556] [Q&A] [answered]
Sorry if this is the wrong category/place for something like this, I'm new to commenting on people's GitHub projects.
Is there a way to take a given first name, last name, and maybe city, and return a list of Facebook users and their IDs so I can get_profile() each of them? Would anyone know an idea of how to do this? Preferably using the cookie so I can be signed in and search.

**Comment by neon-ninja** [comment, marked as answer]
Search isn't implemented yet, see https://github.com/kevinzg/facebook-scraper/issues/419
